### PR TITLE
Refactor Cassandra for less open sessions

### PR DESCRIPTION
**Pull request by wxing1292**
**What changed?**
* Reduce number of open cassandra sessions

**Why?**
See above

**How did you test it?**
N/A

**Potential risks**
N/A

### CODE REVIEW

File: docker/config_template.yaml [hunk 1]
```diff
@@ -37,6 +38,7 @@ persistence:
                 user: {{ default .Env.CASSANDRA_USER "" }}
                 password: {{ default .Env.CASSANDRA_PASSWORD "" }}
                 port: {{ default .Env.CASSANDRA_PORT "9042" }}
+                maxConns: {{ default .Env.CASSANDRA_MAX_CONNS "10" }}
```
Comment (mastermanu) [inline docker/config_template.yaml:41]: This is probably too big. I think we can leave this unset or set it to 2? What do you think

File: docker/config_template.yaml [hunk 2]
```diff
@@ -74,8 +76,8 @@ persistence:
                 connectAttributes:
                     tx_isolation: 'READ-COMMITTED'
                 {{- end }}
-                maxConns: {{ default .Env.SQL_VIS_MAX_CONNS "20" }}
-                maxIdleConns: {{ default .Env.SQL_VIS_MAX_IDLE_CONNS "20" }}
+                maxConns: {{ default .Env.SQL_VIS_MAX_CONNS "10" }}
```
Comment (mastermanu) [inline docker/config_template.yaml:79]: curious, what was the reason for this change?

**Review by mastermanu — APPROVED** [review]

File: docker/config_template.yaml (hunk 2 above)
Comment (wxing1292) [reply docker/config_template.yaml:79]: there is not many calls in visibility side

File: docker/config_template.yaml (hunk 1 above)
Comment (wxing1292) [reply docker/config_template.yaml:41]: i guess 2 is too small

**Pull request merged**